WD-Firmware/rtos/rtosal/api_inc/rtosal_defines.h

```diff
@@ -141,6 +143,17 @@
    #define D_RTOSAL_DONT_START                   TBD_TAKE_VAL_FROM_TX
    #define D_RTOSAL_AUTO_START                   TBD_TAKE_VAL_FROM_TX
    #define D_RTOSAL_NO_TIME_SLICE                TBD_TAKE_VAL_FROM_TX
+#else
+   #error "Add appropriate RTOS definitions"
 #endif /* #ifdef D_USE_FREERTOS */
 
+#ifdef D_USE_FREERTOS
+   #define D_MEPC_LOC_IN_STK    0
```

Comment:
1. convention: D_RTOSAL
2. remove  #define contextSwitchFunc vTaskSwitchContext. tale it to macro in rtoasl_asm_utils.S (also need to change name rtos_utils.S)